frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -184,6 +246,20 @@ pub mod pallet {
 			})
 		}
 
+		#[transactional]
+		fn set_pool_id_for_asset(
+			asset_id: T::AssetId,
+			pool_id: T::PoolId,
+		) -> Result<(), DispatchError> {
+			if Pools::<T>::contains_key(asset_id) {
+				Pools::<T>::mutate(asset_id, |_| pool_id);
```

Comment:
This logic is good, but here is where we need to start the logic of moving assets from one pool to the other.

Reply:
At a high level this makes me think that we need to store the strategies state of each asset.

Reply:
Something like:
`enum State {
Normal,
Transfering
`
`struct PoolConfig {
pool_id,
state: State
}`

and this can be stored in the `Pools` StorageMap